application/src/main/java/com/sanction/thunder/resources/VerificationResource.java

```diff
@@ -144,9 +144,9 @@ public Response createVerificationEmail(@Context UriInfo uriInfo, @Auth Key key,
     boolean emailResult = emailService.sendEmail(result.getEmail(),
         messageOptions.getSubject(),
         EmailUtilities.replaceUrlPlaceholder(messageOptions.getBodyHtml(),
```

Comment:
Is this getting run every time an email is sent? Is there a way to make the replacement once and send that instead of replacing every time? I don't know what kind on impact this might have but it seems like a way to go.

Reply:
I would love to do that but unfortunately the URL is unique per request - it requires the email address of the user sending the email to, and the unique verification token.